src/text/text2.asm

```diff
@@ -1,351 +1,353 @@
-Text013e:
+PrehistoricDreamCheckText: ; $013e
 	textfw "「おもて」なら 「なにかの化石」から"
 	linefw "進化したポケモンの ダメージが+10"
 	done
 
-Text013f:
+PowerOfDarknessCheckText: ; GENGAR_LV40
```

Comment:
These kinds of comments might be a little useful.. we can keep them for now I suppose, but we probably won't keep them forever. Unless you think they're not that helpful for you already then you can get rid of them now.